apps/web/src/app/[locale]/(dashboard)/[wsId]/time-tracker/components/activity-heatmap.tsx

```
@@ -351,8 +548,696 @@
+        </div>
+      </div>
+    );
+  };
+
+  const renderCompactCards = () => {
+    // Group data by month for compact card display
+    const monthlyData = weeks.reduce((acc, week) => {
+      week.forEach(day => {
+        // Add null check for day and day.activity
+        if (day && day.activity) {
+          const monthKey = day.date.format('YYYY-MM');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)